[2024-08-19T22:45:22.461+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-19T22:45:22.668+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [queued]>
[2024-08-19T22:45:22.682+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [queued]>
[2024-08-19T22:45:22.684+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-19T22:45:22.704+0700] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): modify_data> on 2024-08-18 00:00:00+00:00
[2024-08-19T22:45:22.711+0700] {standard_task_runner.py:64} INFO - Started process 27326 to run task
[2024-08-19T22:45:22.716+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'scrap_to_dag', 'modify_data', 'scheduled__2024-08-18T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/oto/scrap_oto.py', '--cfg-path', '/tmp/tmpiccwt5es']
[2024-08-19T22:45:22.720+0700] {standard_task_runner.py:91} INFO - Job 45: Subtask modify_data
[2024-08-19T22:45:22.816+0700] {task_command.py:426} INFO - Running <TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [running]> on host BrightHorizons.
[2024-08-19T22:45:22.977+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrap_to_dag' AIRFLOW_CTX_TASK_ID='modify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-18T00:00:00+00:00'
[2024-08-19T22:45:22.978+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-19T22:45:23.014+0700] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-08-19T22:45:23.015+0700] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py
[2024-08-19T22:45:25.163+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:25 WARN Utils: Your hostname, BrightHorizons resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2024-08-19T22:45:25.167+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:25 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-08-19T22:45:26.987+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:26 INFO SparkContext: Running Spark version 3.5.1
[2024-08-19T22:45:26.988+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:26 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-08-19T22:45:26.989+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:26 INFO SparkContext: Java version 11.0.24
[2024-08-19T22:45:27.111+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-08-19T22:45:27.277+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceUtils: ==============================================================
[2024-08-19T22:45:27.278+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-08-19T22:45:27.279+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceUtils: ==============================================================
[2024-08-19T22:45:27.280+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SparkContext: Submitted application: CarDataAnalysis
[2024-08-19T22:45:27.314+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-08-19T22:45:27.334+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-08-19T22:45:27.338+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-08-19T22:45:27.414+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SecurityManager: Changing view acls to: root
[2024-08-19T22:45:27.415+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SecurityManager: Changing modify acls to: root
[2024-08-19T22:45:27.417+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SecurityManager: Changing view acls groups to:
[2024-08-19T22:45:27.419+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SecurityManager: Changing modify acls groups to:
[2024-08-19T22:45:27.420+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: root; groups with view permissions: EMPTY; users with modify permissions: root; groups with modify permissions: EMPTY
[2024-08-19T22:45:27.836+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO Utils: Successfully started service 'sparkDriver' on port 36079.
[2024-08-19T22:45:27.888+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SparkEnv: Registering MapOutputTracker
[2024-08-19T22:45:27.943+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO SparkEnv: Registering BlockManagerMaster
[2024-08-19T22:45:27.976+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-08-19T22:45:27.977+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:27 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-08-19T22:45:28.030+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-08-19T22:45:28.073+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-44b348e6-e3cc-4bdb-8d86-d56752a52351
[2024-08-19T22:45:28.104+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-08-19T22:45:28.160+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-08-19T22:45:28.387+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-08-19T22:45:28.477+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-08-19T22:45:28.803+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:28 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-08-19T22:45:30.224+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:31.227+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:32.229+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:33.231+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:34.233+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:35.235+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:36.236+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:37.238+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:38.240+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:39.242+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:40.262+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:41.264+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:42.266+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:43.268+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:44.270+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:45.271+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:46.273+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:47.275+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:48.277+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:49.279+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:45:49.281+0700] {spark_submit.py:579} INFO - 24/08/19 22:45:49 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 32419ms.
[2024-08-19T22:46:22.710+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:23.712+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:24.713+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:25.716+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:26.717+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:27.719+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:28.720+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:29.722+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:30.723+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:31.724+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:31.728+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:31 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 23035ms.
[2024-08-19T22:46:55.765+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:56.767+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:57.769+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:58.770+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:46:59.771+0700] {spark_submit.py:579} INFO - 24/08/19 22:46:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:00.773+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:01.775+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:02.779+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:03.781+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:04.783+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:04.787+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:04 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 27556ms.
[2024-08-19T22:47:33.346+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:34.347+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:35.349+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:36.350+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:37.351+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:38.353+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:39.355+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:40.356+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:41.358+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:42.359+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:47:42.361+0700] {spark_submit.py:579} INFO - 24/08/19 22:47:42 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 43184ms.
[2024-08-19T22:48:26.552+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:27.554+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:28.556+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:29.557+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:30.559+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:31.561+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:32.563+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:33.564+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:34.566+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:35.568+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:48:35.572+0700] {spark_submit.py:579} INFO - 24/08/19 22:48:35 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 42083ms.
[2024-08-19T22:49:18.660+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:19.661+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:20.662+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:21.663+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:22.665+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:23.667+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:24.669+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:25.670+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:26.672+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:27.673+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:27.675+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:27 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 6 failover attempts. Trying to failover after sleeping for 28042ms.
[2024-08-19T22:49:56.720+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:57.722+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:58.723+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:49:59.725+0700] {spark_submit.py:579} INFO - 24/08/19 22:49:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:00.728+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:01.730+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:02.731+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:03.733+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:04.734+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:05.736+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:05.738+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:05 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 7 failover attempts. Trying to failover after sleeping for 39273ms.
[2024-08-19T22:50:46.015+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:47.017+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:48.018+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:49.020+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:50.023+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:51.026+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:52.028+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:53.029+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:54.031+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:55.032+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:50:55.035+0700] {spark_submit.py:579} INFO - 24/08/19 22:50:55 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 8 failover attempts. Trying to failover after sleeping for 16503ms.
[2024-08-19T22:51:12.544+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:13.545+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:14.547+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:15.549+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:16.551+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:17.553+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:18.554+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:19.556+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:20.557+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:21.559+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:21.561+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:21 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 9 failover attempts. Trying to failover after sleeping for 15175ms.
[2024-08-19T22:51:37.738+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:38.740+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:39.742+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:40.744+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:41.746+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:42.748+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:43.749+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:44.751+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:45.752+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:46.754+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:51:46.755+0700] {spark_submit.py:579} INFO - 24/08/19 22:51:46 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 10 failover attempts. Trying to failover after sleeping for 22702ms.
[2024-08-19T22:52:10.460+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:11.462+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:12.464+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:13.466+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:14.467+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:15.469+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:16.471+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:17.473+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:18.474+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:19.476+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:19.478+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:19 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 11 failover attempts. Trying to failover after sleeping for 32658ms.
[2024-08-19T22:52:53.138+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:54.140+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:55.142+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:56.143+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:57.145+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:58.147+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:52:59.149+0700] {spark_submit.py:579} INFO - 24/08/19 22:52:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:00.150+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:01.152+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:02.153+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:02.156+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:02 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 12 failover attempts. Trying to failover after sleeping for 39042ms.
[2024-08-19T22:53:42.201+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:43.202+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:44.204+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:45.205+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:46.207+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:47.209+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:48.211+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:49.213+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:50.215+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:51.216+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:53:51.218+0700] {spark_submit.py:579} INFO - 24/08/19 22:53:51 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 13 failover attempts. Trying to failover after sleeping for 41409ms.
[2024-08-19T22:54:33.631+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:34.632+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:35.634+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:36.635+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:37.636+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:38.638+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:39.639+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:40.640+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:41.642+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:42.643+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:54:42.646+0700] {spark_submit.py:579} INFO - 24/08/19 22:54:42 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 14 failover attempts. Trying to failover after sleeping for 24189ms.
[2024-08-19T22:55:07.839+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:08.841+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:09.843+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:10.845+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:11.846+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:12.848+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:13.850+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:14.851+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:15.853+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:16.855+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:16.857+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:16 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 15 failover attempts. Trying to failover after sleeping for 32325ms.
[2024-08-19T22:55:50.186+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:51.187+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:52.189+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:53.191+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:54.192+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:55.195+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:56.197+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:57.198+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:58.199+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:59.200+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:55:59.202+0700] {spark_submit.py:579} INFO - 24/08/19 22:55:59 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 16 failover attempts. Trying to failover after sleeping for 32362ms.
[2024-08-19T22:56:32.567+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:33.568+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:34.570+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:35.571+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:36.572+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:37.574+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:38.576+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:39.577+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:40.579+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:41.581+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:56:41.582+0700] {spark_submit.py:579} INFO - 24/08/19 22:56:41 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 17 failover attempts. Trying to failover after sleeping for 26333ms.
[2024-08-19T22:57:08.918+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:09.919+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:10.921+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:11.922+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:12.923+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:13.925+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:14.927+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:15.929+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:16.930+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:17.932+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:17.933+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:17 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 18 failover attempts. Trying to failover after sleeping for 31007ms.
[2024-08-19T22:57:49.945+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:50.947+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:51.948+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:52.950+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:53.951+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:54.953+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:55.955+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:56.957+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:57.958+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:58.959+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:57:58.962+0700] {spark_submit.py:579} INFO - 24/08/19 22:57:58 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 19 failover attempts. Trying to failover after sleeping for 35644ms.
[2024-08-19T22:58:35.608+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:36.610+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:37.611+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:38.613+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:39.615+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:40.617+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:41.618+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:42.622+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:43.624+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:44.626+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:58:44.628+0700] {spark_submit.py:579} INFO - 24/08/19 22:58:44 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 20 failover attempts. Trying to failover after sleeping for 40703ms.
[2024-08-19T22:59:26.334+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:27.335+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:28.337+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:29.338+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:30.339+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:31.341+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:32.342+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:33.343+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:34.345+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:35.347+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T22:59:35.349+0700] {spark_submit.py:579} INFO - 24/08/19 22:59:35 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 21 failover attempts. Trying to failover after sleeping for 26043ms.
[2024-08-19T23:00:02.394+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:03.396+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:04.398+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:05.399+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:06.401+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:07.402+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:08.404+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:09.406+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:10.407+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:11.409+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:11.411+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:11 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 22 failover attempts. Trying to failover after sleeping for 27147ms.
[2024-08-19T23:00:39.561+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:40.564+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:41.565+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:42.567+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:43.569+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:44.570+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:45.572+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:46.573+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:47.574+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:48.575+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:00:48.577+0700] {spark_submit.py:579} INFO - 24/08/19 23:00:48 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 23 failover attempts. Trying to failover after sleeping for 15876ms.
[2024-08-19T23:01:05.455+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:06.457+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:07.459+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:08.460+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:09.461+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:10.463+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:11.464+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:12.466+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:13.467+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:14.469+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:14.471+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:14 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 24 failover attempts. Trying to failover after sleeping for 19175ms.
[2024-08-19T23:01:34.650+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:35.651+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:36.652+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:37.654+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:38.655+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:39.657+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:40.659+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:41.661+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:42.662+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:43.663+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:01:43.665+0700] {spark_submit.py:579} INFO - 24/08/19 23:01:43 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 25 failover attempts. Trying to failover after sleeping for 24284ms.
[2024-08-19T23:02:08.952+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:09.954+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:10.955+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:11.956+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:12.958+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:13.960+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:14.961+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:15.963+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:16.964+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:17.966+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:02:17.967+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:17 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 26 failover attempts. Trying to failover after sleeping for 40658ms.
[2024-08-19T23:02:59.628+0700] {spark_submit.py:579} INFO - 24/08/19 23:02:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:00.629+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:01.630+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:02.632+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:03.633+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:04.635+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:05.637+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:06.638+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:07.639+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:08.641+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:08.642+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:08 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 27 failover attempts. Trying to failover after sleeping for 18780ms.
[2024-08-19T23:03:28.424+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:29.426+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:30.427+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:31.429+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:32.430+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:33.431+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:34.433+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:35.435+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:36.436+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:37.437+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:03:37.440+0700] {spark_submit.py:579} INFO - 24/08/19 23:03:37 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 28 failover attempts. Trying to failover after sleeping for 39805ms.
[2024-08-19T23:04:18.248+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:19.249+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:20.250+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:21.252+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:22.253+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:23.255+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:24.256+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:25.257+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:26.258+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:27.260+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:04:27.262+0700] {spark_submit.py:579} INFO - 24/08/19 23:04:27 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 29 failover attempts. Trying to failover after sleeping for 40814ms.
[2024-08-19T23:05:09.078+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:10.080+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:11.081+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:12.082+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:13.084+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:14.085+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:15.087+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:16.089+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:17.090+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:18.091+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:05:18.107+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 ERROR SparkContext: Error initializing SparkContext.
[2024-08-19T23:05:18.108+0700] {spark_submit.py:579} INFO - java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort
[2024-08-19T23:05:18.108+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.GeneratedConstructorAccessor28.newInstance(Unknown Source)
[2024-08-19T23:05:18.109+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:05:18.109+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:05:18.110+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-08-19T23:05:18.110+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:824)
[2024-08-19T23:05:18.110+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-08-19T23:05:18.110+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-08-19T23:05:18.110+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy29.getNewApplication(Unknown Source)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.api.impl.pb.client.ApplicationClientProtocolPBClientImpl.getNewApplication(ApplicationClientProtocolPBClientImpl.java:286)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-19T23:05:18.111+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-08-19T23:05:18.112+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-08-19T23:05:18.113+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-08-19T23:05:18.114+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-08-19T23:05:18.114+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-08-19T23:05:18.115+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-08-19T23:05:18.115+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy30.getNewApplication(Unknown Source)
[2024-08-19T23:05:18.116+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.getNewApplication(YarnClientImpl.java:284)
[2024-08-19T23:05:18.116+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.createApplication(YarnClientImpl.java:292)
[2024-08-19T23:05:18.116+0700] {spark_submit.py:579} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:195)
[2024-08-19T23:05:18.119+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2024-08-19T23:05:18.119+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2024-08-19T23:05:18.120+0700] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2024-08-19T23:05:18.121+0700] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-19T23:05:18.121+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-19T23:05:18.122+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2024-08-19T23:05:18.122+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:05:18.122+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:05:18.122+0700] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-19T23:05:18.123+0700] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-19T23:05:18.123+0700] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-19T23:05:18.123+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-19T23:05:18.124+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-19T23:05:18.124+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-19T23:05:18.124+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-19T23:05:18.124+0700] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-08-19T23:05:18.125+0700] {spark_submit.py:579} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-08-19T23:05:18.125+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2024-08-19T23:05:18.126+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2024-08-19T23:05:18.126+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-08-19T23:05:18.126+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-08-19T23:05:18.126+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-08-19T23:05:18.126+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-08-19T23:05:18.127+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-08-19T23:05:18.127+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-08-19T23:05:18.127+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-08-19T23:05:18.127+0700] {spark_submit.py:579} INFO - ... 33 more
[2024-08-19T23:05:18.128+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-08-19T23:05:18.162+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO SparkUI: Stopped Spark web UI at http://10.255.255.254:4040
[2024-08-19T23:05:18.200+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to send shutdown message before the AM has registered!
[2024-08-19T23:05:18.216+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to request executors before the AM has registered!
[2024-08-19T23:05:18.225+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO YarnClientSchedulerBackend: Shutting down all executors
[2024-08-19T23:05:18.235+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO YarnSchedulerBackend$YarnDriverEndpoint: Asking each executor to shut down
[2024-08-19T23:05:18.241+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO YarnClientSchedulerBackend: YARN client scheduler backend Stopped
[2024-08-19T23:05:18.286+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-08-19T23:05:18.301+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO MemoryStore: MemoryStore cleared
[2024-08-19T23:05:18.303+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO BlockManager: BlockManager stopped
[2024-08-19T23:05:18.319+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-08-19T23:05:18.320+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 WARN MetricsSystem: Stopping a MetricsSystem that is not running
[2024-08-19T23:05:18.326+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-08-19T23:05:18.346+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO SparkContext: Successfully stopped SparkContext
[2024-08-19T23:05:18.401+0700] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-08-19T23:05:18.402+0700] {spark_submit.py:579} INFO - File "/home/miracle/mobil/spark/oto.py", line 11, in <module>
[2024-08-19T23:05:18.403+0700] {spark_submit.py:579} INFO - spark = SparkSession.builder.appName("CarDataAnalysis").getOrCreate()
[2024-08-19T23:05:18.403+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 497, in getOrCreate
[2024-08-19T23:05:18.404+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 515, in getOrCreate
[2024-08-19T23:05:18.405+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 203, in __init__
[2024-08-19T23:05:18.405+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 296, in _do_init
[2024-08-19T23:05:18.407+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 421, in _initialize_context
[2024-08-19T23:05:18.408+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2024-08-19T23:05:18.409+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-08-19T23:05:18.409+0700] {spark_submit.py:579} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2024-08-19T23:05:18.409+0700] {spark_submit.py:579} INFO - : java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.GeneratedConstructorAccessor28.newInstance(Unknown Source)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:824)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-08-19T23:05:18.410+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-08-19T23:05:18.411+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-08-19T23:05:18.412+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-08-19T23:05:18.413+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy29.getNewApplication(Unknown Source)
[2024-08-19T23:05:18.413+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.api.impl.pb.client.ApplicationClientProtocolPBClientImpl.getNewApplication(ApplicationClientProtocolPBClientImpl.java:286)
[2024-08-19T23:05:18.413+0700] {spark_submit.py:579} INFO - at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[2024-08-19T23:05:18.413+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-19T23:05:18.413+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-08-19T23:05:18.414+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-08-19T23:05:18.414+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-08-19T23:05:18.414+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-08-19T23:05:18.415+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-08-19T23:05:18.415+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-08-19T23:05:18.415+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy30.getNewApplication(Unknown Source)
[2024-08-19T23:05:18.415+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.getNewApplication(YarnClientImpl.java:284)
[2024-08-19T23:05:18.416+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.createApplication(YarnClientImpl.java:292)
[2024-08-19T23:05:18.416+0700] {spark_submit.py:579} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:195)
[2024-08-19T23:05:18.416+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2024-08-19T23:05:18.416+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2024-08-19T23:05:18.417+0700] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2024-08-19T23:05:18.417+0700] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-19T23:05:18.417+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-19T23:05:18.417+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2024-08-19T23:05:18.418+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:05:18.418+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:05:18.418+0700] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-19T23:05:18.418+0700] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-19T23:05:18.419+0700] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-19T23:05:18.419+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-19T23:05:18.419+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-19T23:05:18.419+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-19T23:05:18.420+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-19T23:05:18.420+0700] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-08-19T23:05:18.420+0700] {spark_submit.py:579} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-08-19T23:05:18.420+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2024-08-19T23:05:18.421+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2024-08-19T23:05:18.421+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-08-19T23:05:18.421+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-08-19T23:05:18.421+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-08-19T23:05:18.422+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-08-19T23:05:18.422+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-08-19T23:05:18.422+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-08-19T23:05:18.422+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-08-19T23:05:18.423+0700] {spark_submit.py:579} INFO - ... 33 more
[2024-08-19T23:05:18.423+0700] {spark_submit.py:579} INFO - 
[2024-08-19T23:05:18.707+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO ShutdownHookManager: Shutdown hook called
[2024-08-19T23:05:18.709+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-7fd25fbc-d7cf-4ae5-952e-c24ecbaa23fc
[2024-08-19T23:05:18.717+0700] {spark_submit.py:579} INFO - 24/08/19 23:05:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-114a1d10-c198-4888-ba8e-cf1a64ff51d8
[2024-08-19T23:05:18.779+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-19T23:05:18.800+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py. Error code is: 1.
[2024-08-19T23:05:18.807+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=scrap_to_dag, task_id=modify_data, run_id=scheduled__2024-08-18T00:00:00+00:00, execution_date=20240818T000000, start_date=20240819T154522, end_date=20240819T160518
[2024-08-19T23:05:18.828+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 45 for task modify_data (Cannot execute: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py. Error code is: 1.; 27326)
[2024-08-19T23:05:18.854+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-19T23:05:18.885+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-19T23:05:18.888+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
